finesse/gui/temp_control.py

```diff
@@ -181,46 +183,52 @@ def _create_controls(self) -> QGridLayout:
         return layout
 
 
-class TC4820_HOT(QGroupBox):
-    """Widgets to view the TC4820 HOT properties."""
+class TC4820(QGroupBox):
+    """Widgets to view the TC4820 properties."""
 
-    def __init__(self) -> None:
-        """Creates the widgets to control and monitor TC4820 HOT.
+    def __init__(self, name: str) -> None:
+        """Creates the widgets to control and monitor a TC4820.
 
-        Returns:
-            None
+        Args:
+            name (str): Name of the blackbody the TC4820 is controlling
         """
-        super().__init__("TC4820 HOT")
+        super().__init__("TC4820 %s" % name.upper())
 
         layout = self._create_controls()
         self.setLayout(layout)
 
     def _create_controls(self) -> QGridLayout:
+        """Creates the overall layout for the panel.
 
+        Returns:
+            QGridLayout: The layout containing the figure.
+        """
         layout = QGridLayout()
 
+        align = Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter
+
         wid = QLabel("CONTROL")
```

Comment:
It's good that you've defined `align` above, but I don't really like that you're reusing `wid`. I'd either give the variables different names or make a separate function for doing this like so:

```py
def _add_label(name: str, row: int, col: int):
```